pkg/kt/snmp.go

```diff
@@ -237,14 +237,26 @@ type SnmpDeviceMetric struct {
 }
 
 const (
-	SNMP_GOOD = 1
-	SNMP_BAD  = 2
+	SNMP_GOOD                   = 1
+	SNMP_BAD_NO_FLOWS           = 3
+	SNMP_BAD_DEVICE_ERR         = 4
+	SNMP_BAD_INTERFACE_ERR      = 5
+	SNMP_BAD_POLL_TIMEOUT       = 6
+	SNMP_BAD_INIT_METADATA      = 7
+	SNMP_BAD_INIT_METRICS       = 8
+	SNMP_BAD_FIRST_METRICS_POLL = 9
 )
 
 var (
 	SNMP_STATUS_MAP = map[int64]string{
-		1: "GOOD",
-		2: "BAD",
+		SNMP_GOOD:                   "GOOD",
+		SNMP_BAD_NO_FLOWS:           "BAD: No flows",
```

Comment:
this language leads me to think my NetFlow flows are broken; but I think the actual measurement is against the amount of data in the response payload from SNMP? 

If that's accurate; I wonder if we should use something more like `SNMP_BAD_NO_DATA: "BAD: No data"`

Reply:
Good call. Updated.